### PR TITLE
Butt Slaps

### PR DESCRIPTION
**Pull request by Sishen1542**
A round of applause!  Same thing as face slapping, but now for your other cheeks. Target groin in disarm intent and click on some spaceman booty. RP has it, main needs it. Roleplay shall never be the same again.

edit:
Butt slaps 2.0
Added nympho to defined traits, added arousal damage to ass slapping (arousal system needs to be enabled), and added forced orgasms to nymphos at max arousal

### CODE REVIEW

**Comment by deathride58** [comment]
Do you think Jay stays away from Citadel, because he, too, is afraid of what he has created?

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1463,6 +1463,19 @@ GLOBAL_LIST_EMPTY(roundstart_races)
...
 		stop_wagging_tail(target)
 		return FALSE
+	var/aim_for_groin  = user.zone_selected == "groin"
+	var/target_aiming_for_groin = target.zone_selected == "groin"
+	if(aim_for_groin && (target_on_help_and_unarmed || target_restrained || target_aiming_for_groin))
+		playsound(target.loc, 'sound/weapons/slap.ogg', 50, 1, -1)
+		user.visible_message("<span class='danger'>[user] slaps [target]'s ass!</span>",
+			"<span class='notice'>You slap [target]'s ass! </span>",\
+		"You hear a slap.")
+		stop_wagging_tail(target)
```
Comment (deathride58) [inline code/modules/mob/living/carbon/human/species.dm]: make this only stop wagging the tail if the target isn't a nymphomaniac

File: code/__DEFINES/traits.dm [hunk 2]
```diff
@@ -70,6 +70,7 @@
 #define	TRAIT_PHOTOGRAPHER		"photographer"
 #define	TRAIT_MUSICIAN			"musician"
 #define	TRAIT_CROCRIN_IMMUNE    "crocin_immune"
+#define TRAIT_NYMPHO			"nymphomania"
```
Comment (deathride58) [inline code/__DEFINES/traits.dm:73]: unused trait? you havent made anything in the code use this

**Review by deathride58 — DISMISSED** [review]

File: code/__DEFINES/traits.dm (hunk 2 above)
Comment (Sishen1542) [reply code/__DEFINES/traits.dm:73]: Couldn't figure out how to force orgasm with nymphos only without adding it in the define.

File: code/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Sishen1542) [reply code/modules/mob/living/carbon/human/species.dm]: Added, also added the nympho check for wagging to face slapping.

**Review by silicons — APPROVED** [review]

**Pull request merged**